here-naksha-lib-base/src/jvmMain/kotlin/naksha/base/Platform.kt

```diff
@@ -374,7 +391,7 @@ actual class Platform {
                     copy as T
                 }
                 is JvmList -> {
-                    val copy = JvmList(obj.size)
+                    val copy = JvmList()
```

Comment:
Why did you removed the size hint? The `JvmList` is backed by an `ArrayList` and I improves the algorithm a lot to add the size hint here?

Reply:
the constructor does not take in the size, it takes in the vararg elements. In one manual test, the size was actually added as another element

Reply:
<img width="461" alt="image" src="https://github.com/user-attachments/assets/a0f8e73f-7f67-4aef-947e-906e85252dd3">

Reply:
True, my fault, but we need to call the capacity hinting (`setCapacity`).

Reply:
Will `setCapacity` in this branch, so please don't change it in other branches to avoid merge conflicts :)